### PR TITLE
Remove Milvus.Client dependency.

### LINKED ISSUE #6431: Aspire.Hosting.Milvus shipped stable in 8.2 but is now shipping preview in 9.0
Aspire.Hosting.Milvus already shipped stable in 8.1 and 8.2. 

However, https://github.com/dotnet/aspire/pull/5707 added a dependency from Aspire.Hosting.Milvus to Milvus.Client. Milvus.Client doesn't have a stable version yet.

https://github.com/dotnet/aspire/pull/6388 marked this package as `SuppressFinalPackageVersion=true` for this reason.

If we don't ship a "stable" 9.0.0 package, people will continue using the 8.2.1 Aspire.Hosting.Milvus package with Aspire 9.0 because that is the latest "stable" version.

We need to decide what to do here for 9.0.0.

cc @mitchdenny @davidfowl @timheuer @DamianEdwards @Alirexaa

### PR DESCRIPTION
**Pull request by mitchdenny**
## Description

Removing Milvus client dependency (incl health check) so we can ship the hosting package as GA.

Fixes #6431

## Checklist

- Is this feature complete?
  - [x] Yes. Ready to ship.
  - [ ] No. Follow-up changes expected.
- Are you including unit tests for the changes and scenario tests if relevant?
  - [ ] Yes
  - [x] No (removed a unit test)
- Did you add public API?
  - [ ] Yes
    - If yes, did you have an API Review for it?
      - [ ] Yes
      - [ ] No
    - Did you add `<remarks />` and `<code />` elements on your triple slash comments?
      - [ ] Yes
      - [ ] No
  - [x] No
- Does the change make any security assumptions or guarantees?
  - [ ] Yes
    - If yes, have you done a threat model and had a security review?
      - [ ] Yes
      - [ ] No
  - [x] No
- Does the change require an update in our Aspire docs?
  - [ ] Yes
    - Link to aspire-docs issue: 
  - [x] No

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/6435)

### CODE REVIEW

File: src/Aspire.Hosting.Milvus/MilvusBuilderExtensions.cs [hunk 1]
```diff
@@ -56,27 +54,6 @@ public static IResourceBuilder<MilvusServerResource> AddMilvus(this IDistributed
...
-        {
-            var connectionString = await milvus.ConnectionStringExpression.GetValueAsync(ct).ConfigureAwait(false)
-            ?? throw new DistributedApplicationException($"ConnectionStringAvailableEvent was published for the '{milvus.Name}' resource but the connection string was null.");
-            milvusClient = CreateMilvusClient(@event.Services, connectionString);
-        });
-
-        var healthCheckKey = $"{name}_check";
-        // TODO: Use health check from AspNetCore.Diagnostics.HealthChecks once it's implemented via this issue:
-        // https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/issues/2214
-        builder.Services.AddHealthChecks()
```
Comment (eerhardt) [inline src/Aspire.Hosting.Milvus/MilvusBuilderExtensions.cs:72]: What other Hosting integration packages don't have health checks? Will this one be unique?

Comment (timheuer) [reply src/Aspire.Hosting.Milvus/MilvusBuilderExtensions.cs:72]: No it will not be unique. Seq, Qdrant

**Review by eerhardt — APPROVED** [review]
I think this is the right change for now. Hopefully Milvus.Client releases stable soon.

**Comment by Alirexaa** [comment]
We should log an issue for Adding the health check to Milvus.

**Pull request merged**